src/pages/Upstream/components/Step1.tsx

```diff
@@ -171,6 +204,390 @@ const Step1: React.FC<Props> = ({ form, disabled }) => {
 
   const renderTimeUnit = () => <span style={{ margin: '0 8px' }}>ms</span>;
 
+  function activeChange() {
```

Comment:
We'd better use `const fn = () => {}` to have a unified style :D

Reply:
Also, for onChange handler, I would prefer using some name like `handleXXX`

Reply:
done